packages/clay-color-picker/src/index.tsx

```diff
@@ -149,6 +149,14 @@ const ClayColorPicker: React.FunctionComponent<IProps> = ({
 	value = 'FFFFFF',
 	...otherProps
 }: IProps) => {
+	if (value.indexOf('#') === 0) {
+		value = value.slice(1);
+	}
+
+	if (useNative && window.navigator.userAgent.indexOf('MSIE') !== -1) {
```

Comment:
Instead of sniffing for IE, wouldn't using feature detection be better? Something like:
```
var input = document.createElement('input');
input.setAttribute('type', 'color');
var inputColorSupport = input.value === '' || true;

if (inputColorSupport) {
    useNative = false;
}
```

Reply:
Good idea, just updated